lib/mapbox/geocoder.rb

```diff
@@ -6,17 +6,27 @@ class Geocoder
     include Mapbox::APIOperations::Request
     extend Mapbox::HashUtils
 
-    def self.geocode_forward(query, proximity=nil, dataset='mapbox.places')
-      proximity_param = ''
+    def self.geocode_forward(query, options={}, dataset='mapbox.places')
+      proximity = options[:proximity] || nil
```

Comment:
Yes, I was trying to keep changes to a minimum and this avoided an undefined method error if `proximity = false`, but I can simplify things and just check `if proximity` instead.